### PR TITLE
Destory the element and reset the Stripe object

### PR DESCRIPTION
**Pull request by jasonlfunk**
This code makes the code destroy the element and reset the Stripe object when the element is unloaded.

The destroy() is needed to prevent the "IntegrationError: Can only create one Element of type card".

Resetting the Stripe object is necessary to have Stripe re-initialized on the next component mount. This is necessary in order to load remote fonts (and probably other reasons as well.)

This PR isn't throughly tested but it is working fine in my environment.

### CODE REVIEW

File: src/stripeElements.js [hunk 1]
```diff
@@ -49,3 +49,9 @@ export function create(elementType, key_or_stripe, options = {}) {
 
   return element
 }
+
+export function destroy() {
+    Stripe.instance = null;
+    Stripe.elements = null;
+    Stripe.createToken = null;
```
Comment (softbeehive) [inline src/stripeElements.js]: Can you remove semicolons?

File: src/StripeElement.vue [hunk 2]
```diff
@@ -26,6 +26,8 @@ export default {
 
   beforeDestroy () {
     this._element.unmount()
+    this._element.destroy()
+    destroy();
```
Comment (softbeehive) [inline src/StripeElement.vue]: Please remove semicolon

**Comment by morrislaptop** [comment]
Is this supposed to work when hot reloading?

**Comment by lisa13** [comment]
This is the solution for Angular 2+ :
Add OnDestroy Lifecycle hook.

 ` ngOnDestroy(): void {
    this.card.unmount();
    this.card.destroy();
  }`

**Pull request merged**